Couchers-org/couchers
# Change switches to new custom one

This is just some tidying. Now that we have a custom switch that allows custom color and loading state, I switched out our other two switches to the new one. To be honest they don't really need loading state but at least it's consistent. I also made the custom switch more flexible for reuse.

**TO TEST**:

- Go to events page, make two events, one canceled and one not. Make sure the switch still filters.
- Open incognito window (or log out), go to login page. Add a console log in the `LoginForm` handleSubmit function. Make sure the value is changing when clicking the switch and hitting submit.
- Make sure notification settings switches still work.

**Web frontend checklist**
- [ x ] Formatted my code with `yarn format`
- [ x ] There are no console warnings when running the app
- [ N/A ] Added any new components to storybook
- [ N/ A] Added tests where relevant
- [ x ] Clicked around my changes running locally and it works
